website/www/site/content/en/contribute/_index.md

```diff
@@ -21,55 +24,61 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Beam Contribution Guide
+# Contribution guide
+
+<a class="arrow-list-header" data-toggle="collapse" href="#collapseOverview" role="button" aria-expanded="false"        aria-controls="collapseOverview">
+   {{< figure src="/images/arrow-icon_list.svg">}}
```

Comment:
Can you confirm that these expand buttons are working as intended? The behavior seems odd to me.
On desktop: 
- Content is always expanded, no arrow shows up. This could be intentional, but also ...
- The headings look like links, and when I hover the underline goes away, but its not actually clickable.

On mobile:
- The contributing code section only collapses to down to "To install these in a debian-based distribution"

Reply:
Yes it was all intentional, only form of the headings are to be fixed

Reply:
Ok the dropdown buttons work better on desktop now. They still seem broken on mobile though, nothing happens when I click them.

Reply:
okay I will check it out

Reply:
Ok the dropdowns seem to be working now. But the second one still only collapses down to "To install these in a debian-based distribution", which doesn't seem correct. That's still part of the Contributing Code section

Reply:
It looks like there's a <div> that just needs to be moved lower.

Reply:
It was designed that way because there was less text, so for now I moved the `div` lower

Reply:
Thank you